prql-python/python/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+import yaml
+
+
+@pytest.fixture()
+def example_queries():
+    book_path = "../website/content/_index.md"
```

Comment:
Technically these aren't actually in the book; they're only on the website...

```suggestion
    website_path = "../website/content/_index.md"
```